libs/@guardian/libs/src/datetime/timeAgo.ts

```diff
@@ -65,33 +65,55 @@ export const timeAgo = (
 	const length = Math.round(rawLength);
 
 	// Dates in the future are not supported
-	if (length < 0) return false;
+	if (length < 0) {
+		return false;
+	}
 
 	switch (unit) {
 		case 'second': {
-			if (length > 55) return verbose ? '1 minute ago' : '1m ago';
-			if (length < 15) return 'now';
-			if (!verbose) return `${length}s ago`;
+			if (length > 55) {
+				return verbose ? '1 minute ago' : '1m ago';
+			}
+			if (length < 15) {
+				return 'now';
+			}
+			if (!verbose) {
+				return `${length}s ago`;
+			}
 			return `${length} seconds ago`;
 		}
 		case 'minute': {
-			if (length > 55) return verbose ? '1 hour ago' : '1h ago';
-			if (!verbose) return `${length}m ago`;
-			if (length == 1) return '1 minute ago';
+			if (length > 55) {
+				return verbose ? '1 hour ago' : '1h ago';
+			}
+			if (!verbose) {
+				return `${length}m ago`;
+			}
+			if (length == 1) {
+				return '1 minute ago';
+			}
```

Comment:
I think these would look nicer with an `else if`, now… might address in a follow-up

Reply:
#1376